main.go

```diff
@@ -124,6 +129,11 @@ func main() {
 		panic(err)
 	}
 
+	err = signScorecardResult(scorecardResultsFile)
+	if err != nil {
+		panic(err)
```

Comment:
Let's not `panic` here. Upload to Rekor may fail and we don't want that to show up as failures to users. Instead let's do this - log this error and quietly return. In a future PR when you add the POST call to the API, make the error a part of the POST payload. On the API side, we can log these errors so that we can monitor these errors through there.

@asraa fyi who brought up this concern.